@ecomplus/widget-offers-notification/cms.config.js

```diff
@@ -0,0 +1,41 @@
+import { name } from './package.json'
+
+export default () => ({
+  label: 'Notificações de Ofertas',
+  fields: [
+    {
+      name: 'pkg',
+      widget: 'hidden',
+      default: name
+    },
+    {
+      label: 'Ativar',
+      hint: 'Habilitar Notificações de Ofertas',
+      name: 'active',
+      widget: 'boolean'
+    },
+    {
+      label: 'Me Avise',
+      hint: 'Habilitar opção `Me Avise` na página de produtos com estoque zerado',
+      name: 'enabledOutOfStock',
+      widget: 'boolean'
+    },
+    {
+      label: 'Acompanhar Preço',
+      hint: 'Habilitar opção `Acompanhar Preço` na página de produtos',
+      name: 'enabledPriceChange',
+      widget: 'boolean'
+    },
```

Comment:
Configuração do widget dentro do objeto `options`:

```suggestion
    {
      label: 'Opções',
      name: 'options',
      widget: 'object',
      hint: 'Personalizações do widget',
      fields: [
        {
          label: 'Me Avise',
          hint: 'Habilitar opção `Me Avise` na página de produtos com estoque zerado',
          name: 'enabledOutOfStock',
          widget: 'boolean'
        },
        {
          label: 'Acompanhar Preço',
          hint: 'Habilitar opção `Acompanhar Preço` na página de produtos',
          name: 'enabledPriceChange',
          widget: 'boolean'
        }
      ]
    },
```